sale.py

```diff
@@ -371,7 +371,10 @@ def create_using_magento_data(cls, order_data):
             # and let the user fix this manually.
             MagentoException.create([{
                 'origin': '%s,%s' % (sale.__name__, sale.id),
-                'log': e.message
+                'log': '''
+                    Error occurred on transitioning to state %s.
+                    Error Message: %s
+                ''' % (order_data['state'], e.message)
```

Comment:
`order_data['state']` will be of magento. better use tryton state

`tryton_state = MagentoOrderState.get_tryton_state(order_data['state'])`